src/index.html

```diff
@@ -7,10 +7,29 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>Search bar by vmisai</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="search_form form_big">
+      <label class="search_label search_label_big">
+        <input
+          data-qa="keypress"
+          class="search_input search_input_big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form data-qa="small" class="search_form form_small">
+      <label class="search_label search_label_small">
+        <input
+          class="search_input search_input_small"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
```

Comment:
let's try to use BEM naming 
```suggestion
    <form data-qa="small" class="search search--small">
      <label class="search__label search__label--small">
        <input
          class="search__input search__input--small"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
```
https://en.bem.info/methodology/quick-start/